packages/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -75,6 +69,15 @@ const ALERT_RULE_TYPE_ID = `${ALERT_RULE_NAMESPACE}.rule_type_id` as const;
 const ALERT_RULE_UPDATED_AT = `${ALERT_RULE_NAMESPACE}.updated_at` as const;
 const ALERT_RULE_UPDATED_BY = `${ALERT_RULE_NAMESPACE}.updated_by` as const;
 const ALERT_RULE_VERSION = `${ALERT_RULE_NAMESPACE}.version` as const;
+// the feature instantiating a rule type.
+// Rule created in stack --> alerts
+// Rule created in siem --> siem
+const ALERT_CONSUMER = `${ALERT_RULE_NAMESPACE}.consumer` as const;
+// the plugin that registered the rule type.
+// Rule type apm.error_rate --> apm
+// Rule type siem.signals --> siem
+const ALERT_PRODUCER = `${ALERT_RULE_NAMESPACE}.producer` as const;
```

Comment:
by convention these would be `ALERT_RULE_CONSUMER` and `ALERT_RULE_PRODUCER` since they're in the `ALERT_RULE_NAMESPACE` right?

Reply:
Are you suggesting I rename all the renames?! 😱  jk. Yea, you're totally right, updating.